beast/physicsmodel/prior_weights_stars.py

```diff
@@ -83,7 +87,8 @@ def compute_age_prior_weights(logages, age_prior_model):
         )
 
     # normalize to avoid numerical issues (too small or too large)
-    age_weights /= np.average(age_weights)
+    # do not normalize as the absolute level now matters for simulations and megaBEAST
+    # age_weights /= np.average(age_weights)
```

Comment:
Since normalization was originally to avoid numerical issues, do we need to worry about that now?

Reply:
I just did a blanket normalization to all the priors in multiple places.  I checked and it should not be an issue for any reasonable age prior.